worldengine/world.py

```diff
@@ -351,7 +350,7 @@ def sea_level(self):
     # Tiles around
     #
 
-    def on_tiles_around_factor(self, factor, pos, radius=1, action=None):
+    def on_tiles_around_factor(self, factor, pos, action, radius=1):
```

Comment:
I have, and they were all called using the keyword.
Although I didn't find a call to ```on_tiles_around```.